README.rst

```diff
@@ -21,6 +21,8 @@ Please use the `main ZMON repository`_ to start a Vagrant demo box.
 
 Make sure the provided Vagrant-Box is up and all services are running.
 
+Update your GitHub ``client-id`` and ``client-secret`` in ``zmon-controller-app/src/main/config/application-github.yml``.
```

Comment:
Yeah, I already removed that in the other PR. Looks like I based this one on top if the other ;/